app/controllers/users_controller.rb

```diff
@@ -5,9 +5,10 @@ class UsersController < ApplicationController
   GROUP_MESSAGING_ENABLED = "1"
 
   before_action :set_user, only: %i[show edit update]
+  before_action :authenticate_user!
```

Comment:
Notice that we now force uses to be authenticated before getting to this page.

Reply:
@hectorcorrea is this a timing thing?  Aka the authenticate_user! has to happen after the set_user?

Reply:
@carolyncole I don't think so, I suspect it could go before it too. Would that make more sense?

Reply:
From slack: We are now suspectingthere may be a timing issue at play and we will leave it in palce here